pyblish/lib.py

```diff
@@ -349,3 +351,27 @@ def _resolve_name(name, package, level):
     __import__(name)
 
     return sys.modules[name]
+
+
+def emit(signal, **kwargs):
+    """Searches through the registered callbacks and executes any callback that
+    matches the signal.
+    The keyword arguments (kwargs) are passed directly through from the emitter
+    to the callbacks.
+
+        Arguments:
+            signal (string): Name of signal emitted
+
+        Example:
```

Comment:
When you make an example using `>>>` in a docstring, it qualifies as a "doctest". Meaning it'll get run during testing and what looks like the output in the example will be the assertion it makes. In this case, the output is expected to be `{'something': 'cool'}` so if you put that underneath the `>>>` like it would have appeared in a terminal, it will indicate failure during testing if it ever goes bad. :)

- https://docs.python.org/2/library/doctest.html